server/index.ts

```diff
@@ -44,14 +44,15 @@ SuperTokens.init({
 	},
 	recipeList: [
 		Dashboard.init({
-			apiKey: "someapikey",
+			apiKey: "",
```

Comment:
This should be optional, so remove it to make sure it wont throw errors without it